app/Utils/ClientPortal/CustomMessage/CustomMessage.php

```diff
@@ -82,12 +82,12 @@ private function compose(): array
             '$company.name' => optional($this->company)->getSetting('name'),
             '$company.website' => optional($this->company)->getSetting('website'),
 
-            '$client.id' => optional(optional($this->client)->present())->id,
-            '$client.name' => optional(optional($this->client)->present())->name,
```

Comment:
it makes sense to leaver the ->present() on the name field as there is additional logic with helper that generate an alternate client.name if the property itself hasn't been filled.